installer/templates/phx_web/templates/layout/root.html.heex

```diff
@@ -1,14 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
-    <meta name="csrf-token" content={get_csrf_token()}>
-    <.live_title suffix=" · Phoenix Framework"><%%= assigns[:page_title] || "<%= @app_module %>" %></.live_title>
-    <link phx-track-static rel="stylesheet" href={~p"/assets/app.css"}/>
-    <script defer phx-track-static type="text/javascript" src={~p"/assets/app.js"}></script>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="csrf-token" content={get_csrf_token()} />
+    <.live_title suffix=" · Phoenix Framework">
+      <%%= assigns[:page_title] || "<%= @app_module %>" %>
+    </.live_title>
+    <link phx-track-static rel="stylesheet" href={~p"/assets/app.css"} />
+    <script defer phx-track-static type="text/javascript" src={~p"/assets/app.js"}>
+    </script>
   </head>
   <body class="bg-white antialiased">
+    <.flash kind={:info} title="Success!" flash={@flash} />
```

Comment:
Added the flash component here to make this test pass https://github.com/phoenixframework/phoenix/blob/16c3e9e818be9122ea6b666f6beecf11c4548fd1/priv/templates/phx.gen.auth/reset_password_controller_test.exs#L118-L119

It gets redirected to `/` and it won't display flash from template `app` due to https://github.com/phoenixframework/phoenix/blob/16c3e9e818be9122ea6b666f6beecf11c4548fd1/installer/templates/phx_web/controllers/page_controller.ex#L4